cogboard-app/src/test/resources/com/cognifide/cogboard/config/credentials-test.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
merge `credentials-test.json` and `endpoints-test.json` so it is presented as in actual code.